mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/AddressBookScreenV2.tsx

```diff
@@ -0,0 +1,458 @@
+                    title: 'Address Details',
+                    isAddNew: false,
+                    address: item.address,
+                    addressLabel: item,
+                    onSaveButtonPress: () => {}
+                  },
+                  merge: true
+                })
+              } else {
+                await openURL(getAddressUrl(item.address))
+              }
+            }}
+            style={tailwind('flex flex-row items-center flex-auto')}
+          >
+            <View style={tailwind('flex flex-auto mr-1')}>
+              {item.label !== '' &&
+                <ThemedTextV2
+                  style={tailwind('font-semibold-v2 text-sm')}
+                  testID={`address_row_label_${index}_${testIDSuffix}`}
+                >
+                  {item.label}
+                </ThemedTextV2>}
+              <ThemedTextV2
+                style={tailwind('font-semibold-v2 text-xs w-10/12 mt-1')}
+                light={tailwind('text-mono-light-v2-700')}
+                dark={tailwind('text-mono-dark-v2-700')}
+                ellipsizeMode='middle'
+                numberOfLines={1}
+                testID={`address_row_text_${index}_${testIDSuffix}`}
+              >
+                {item.address}
+              </ThemedTextV2>
+            </View>
+            <ThemedIcon
+              dark={tailwind('text-mono-dark-v2-700')}
+              light={tailwind('text-mono-light-v2-700')}
+              iconType='Feather'
+              name={activeButtonGroup === ButtonGroupTabKey.Whitelisted ? 'chevron-right' : 'external-link'}
+              size={18}
+            />
+          </TouchableOpacity>
+        </View>
+      </ThemedViewV2>
+    )
+  }, [filteredAddressBook, filteredWalletAddress, activeButtonGroup])
+
+  const goToAddAddressForm = (): void => {
+    navigation.navigate({
+      name: 'AddOrEditAddressBookScreen',
+      params: {
+        title: 'Add Address',
+        isAddNew: true,
+        onSaveButtonPress: (address?: string) => {
+          if (onAddressSelect !== undefined && address !== undefined) {
+            onAddressSelect(address)
+          }
+        }
+      },
+      merge: true
+    })
+  }
+
+  return (
+    <ThemedViewV2
+      style={tailwind('h-full')}
+    >
+      <ThemedViewV2
+        light={tailwind('bg-mono-light-v2-00 border-mono-light-v2-100')}
+        dark={tailwind('bg-mono-dark-v2-00 border-mono-dark-v2-100')}
+        style={tailwind('flex flex-col items-center pt-1 rounded-b-2xl border-b')}
+      >
+        <View style={tailwind('w-full px-5')}>
+          <ButtonGroupV2
+            buttons={buttonGroup}
+            activeButtonGroupItem={activeButtonGroup}
+            testID='address_button_group'
+            lightThemeStyle={tailwind('bg-transparent')}
+            darkThemeStyle={tailwind('bg-transparent')}
+          />
+        </View>
+      </ThemedViewV2>
+      <ScrollView
+        contentContainerStyle={tailwind('pb-16')}
```

Comment:
I think 64px create too much empty space, can we use 32px to match the padding above search input?
```suggestion
        contentContainerStyle={tailwind('pb-8')}
```